#########################################################
kallisto pipeline version 1.0.0
Wrote by m.kouhsar@exeter.ac.uk
R script for running DEG analysis using the Sleuth package
#########################################################
Warning message:
In dir.create(path = dirname(OutPrefix), recursive = T) :
  '/lustre/projects/Research_Project-191391/Morteza/kallisto' already exists
Input arguments:
        kallisto results directory: /lustre/projects/Research_Project-191391/Morteza/kallisto/BDR/
        Phenotype file: /lustre/projects/Research_Project-191391/Morteza/kallisto/BDR.Phenotype.Labeled.csv
        Target mapping file: /lustre/projects/Research_Project-191391/Morteza/kallisto/Human.Transcript.Ensemble.GRCh38.txt
        Factor variables in the model: Phenotype, Gender
        Numeric variables in the model: Age, RIN
        Removing Outliers: yes
        Output files prefix: /lustre/projects/Research_Project-191391/Morteza/kallisto/BDR
        Script directory: /lustre/projects/Research_Project-191391/Morteza/github/RNA-Seq-analysis/Mapping-to-Ref-Genome/kallisto/
#########################################################
reading in kallisto results
dropping unused factor levels
.............................................................
normalizing est_counts
85616 targets passed the filter
normalizing tpm
merging in metadata
summarizing bootstraps
................................................
.............
Saving Sleuth Object in /lustre/projects/Research_Project-191391/Morteza/kallisto/BDR.SO.rdat ...
Warning messages:
1: In sleuth_prep(pheno, target_mapping = target_map, extra_bootstrap_summary = TRUE) :
  Your 'sample_to_covariates' data.frame contains NA values. This will likely cause issues later.
2: In check_target_mapping(tmp_names, target_mapping, !is.null(aggregation_column)) :
  intersection between target_id from kallisto runs and the target_mapping is empty. attempted to fix problem by removing .N from target_id, then merging back into target_mapping. please check obj$target_mapping to ensure this new mapping is correct.
Warning message:
package ‘ggplot2’ was built under R version 4.4.3 
Finding Outlier samples using Mahalanobis distance and chi-squared distribution...
5 outliers found:
BBN15247, BBN15251, BBN20618, BBN00328569, BBN00426227
Saving Outlier plots..
Removing outliers and updating SO object...
reading in kallisto results
dropping unused factor levels
........................................................
normalizing est_counts
85364 targets passed the filter
normalizing tpm
merging in metadata
summarizing bootstraps
........................................................

Saving updated Sleuth Object in /lustre/projects/Research_Project-191391/Morteza/kallisto/BDR.SO.rdat ...
Warning messages:
1: In geom_point(aes(center_[1], center_[2]), size = 5, color = "blue") :
  All aesthetics have length 1, but the data has 61 rows.
ℹ Please consider using `annotate()` or provide this layer with data containing
  a single row.
2: In sleuth_prep(pheno.so, target_mapping = target_map, extra_bootstrap_summary = TRUE) :
  Your 'sample_to_covariates' data.frame contains NA values. This will likely cause issues later.
3: In check_target_mapping(tmp_names, target_mapping, !is.null(aggregation_column)) :
  intersection between target_id from kallisto runs and the target_mapping is empty. attempted to fix problem by removing .N from target_id, then merging back into target_mapping. please check obj$target_mapping to ensure this new mapping is correct.
All done!
